### PR TITLE
Doc: Set up redirects for deleted pages

### PR DESCRIPTION
**Pull request by karenzone**
**PREVIEWS:** 
* https://logstash_13772.docs-preview.app.elstc.co/guide/en/logstash/master/brew.html
* https://logstash_13772.docs-preview.app.elstc.co/guide/en/logstash/master/upgrading-logstash-pqs.html

This PR sets up links to deleted pages so that we don't give our users a bad 404 experience (if they come into docs from a saved link or from a web search).

### CODE REVIEW

File: docs/static/redirects.asciidoc [hunk 1]
```diff
@@ -28,13 +30,79 @@ If your use case involves reading files that contain multiline entries,
...
 
 As of Logstash 8.0, Elastic no longer maintains a homebrew cask containing formulae for installing the Elastic-licensed distribution of Logstash.
 If you want to run the full distribution of Logstash on a Mac, you are encouraged to <<installing-binary,install from a downloaded binary distribution>>.
 
 You can still install the Apache-licensed OSS distribution with homebrew using the formulae maintained by Homebrew.
+
+[role="exclude",id="brew-start"]
+==== Homebrew for Logstash
+
+As of Logstash 8.0, Elastic no longer maintains a homebrew cask containing formulae for installing the Elastic-licensed distribution of Logstash.
```
[the remark this answers is not in the document]
Comment (karenzone) [reply docs/static/redirects.asciidoc:47]: The content has been relocated to a deleted pages section that is not indexed (hence `[role="exclude"]` in the anchor] and not listed in the TOC.   

You can tell that something different is going on here if you check out the breadcrumbs: 
Screen Shot 2022-02-16 at 5 46 45 PM

This approach lets us gracefully remove older content from the body of the guide without causing 404's. Any saved link or old link from an internet search can still resolve.

**Comment by karenzone** [comment]
> Overall seems ok, left a note on a typo plus a doubt on how this changes impact positively in the 404 problem

Answered here: https://github.com/elastic/logstash/pull/13772#discussion_r808520953
(Reposting because the Q/A are hidden as "resolved."

File: docs/static/redirects.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (andsel) [reply docs/static/redirects.asciidoc]: :+1 thanks @karenzone for explaining :-)

**Review by andsel — APPROVED** [review]
LGTM

**Pull request merged**